shuttle-explorer/README.md

```diff
@@ -0,0 +1,34 @@
+# Shuttle Explorer
+
+Timeline visualiser for [Shuttle](https://github.com/awslabs/shuttle).
+
+## Building
+
+To use the extension, it is currently required to build it from source. Pre-requisites include:
+
+- VS Code 1.92.0 or later
+- `node` 18.20 and `npm` 10.5 (other versions are probably fine too)
+- [esbuild Problem Matcher extension](https://marketplace.visualstudio.com/items?itemName=connor4312.esbuild-problem-matchers), installed either globally or in the Shuttle Explorer workspace. **Without this extension the builds will fail silently!**
```

Comment:
Why do we need this?

Reply:
As explained: without the "problem matcher", VS Code does not really understand that a background task is a background task. Some additional context [here](https://stackoverflow.com/a/49034830). This extension is also recommended by the VS Code guide for using `esbuild` for building extensions, [here](https://code.visualstudio.com/api/working-with-extensions/bundling-extension#run-esbuild).